testing_and_setup/compass/ocean/global_ocean/scripts/create_E3SM_coupling_files.py

```diff
@@ -0,0 +1,309 @@
+    vertCoordMovementWeights,boundaryLayerDepth,seaIcePressure,\
+    atmosphericPressure,filteredSSHGradientZonal,filteredSSHGradientMeridional',
+            '-O',  # Overwrite existing file
+            mesh_name + '.nc',
+            'seaice.' + mesh_name + '.nc'
+            ]
+    run_command(args)
+
+    # make links to output directory
+    os.chdir('../assembled_files_for_upload/input_data/ice/mpas-cice/' + mesh_name)
+    make_link( '../../../../../initial_condition_seaice/seaice.' + mesh_name + '.nc',
+               'seaice.' + mesh_name + '.nc')
+
+
+def scrip(config, date_string):
+
+    # obtain configuration settings
+    mesh_name = config.get('main', 'mesh_name')
+
+    # name scrip files
+    scrip_file = ('ocean.' + mesh_name + '.scrip.' + date_string + '.nc')
+
+    # create links
+    make_link('../init.nc', mesh_name + '.nc')
+
+    # command line execution
+    args = [
+        'create_SCRIP_file_from_MPAS_mesh.py',
+        '-m', mesh_name + '.nc',
+        '-s', scrip_file]
+    run_command(args)
+
+    # make links to output directories
+    os.chdir('../assembled_files_for_upload/input_data/ocn/mpas-o/' + mesh_name)
+    make_link('../../../../../scrip/' + scrip_file, scrip_file)
+
+
+def transects_and_regions(config, date_string):
+
+    # obtain configuration settings
+    mesh_name = config.get('main', 'mesh_name')
+    transect_region_geojson = config.get('transects_and_regions', 'transect_region_geojson')
+
+    maskfile = 'masks_SingleRegionAtlanticWTransportTransects.' + mesh_name + '.nc'
+
+    # make links
+    make_link('../init.nc', mesh_name + '.nc')
+    make_link(transect_region_geojson,
+        'SingleRegionAtlanticWTransportTransects.geojson')
+
+    # check: how does this call MpasMaskCreator?
+    args = ['MpasMaskCreator.x', mesh_name + '.nc', maskfile,
+            '-f', 'SingleRegionAtlanticWTransportTransects.geojson']
+    run_command(args)
+
+    # make links in output directory
+    os.chdir('../assembled_files_for_upload/input_data/ocn/mpas-o/' + mesh_name)
+    make_link('../../../../../transects_and_regions/masks_SingleRegionAtlanticWTransportTransects.' + mesh_name + '.nc',
+             'masks_SingleRegionAtlanticWTransportTransects.' + mesh_name + '.nc')
+
+
+def mapping_Gcase(config, date_string):
+
+    # obtain configuration settings
+    mesh_name = config.get('main', 'mesh_name')
+    atm_scrip_tag = 'T62_040121'
+    atm_scrip_path = config.get('mapping_Gcase', 'atm_scrip_path')
+
+    # make links
+    ocn_scrip_file = 'ocean.' + mesh_name + '.scrip.' + date_string + '.nc'
+    make_link('../scrip/ocean.' + mesh_name + '.scrip.' + date_string + '.nc',
+              ocn_scrip_file)
+    atm_scrip_file = atm_scrip_tag + '.nc'
+    make_link(atm_scrip_path + '/' + atm_scrip_file, atm_scrip_file)
+
+    # execute commands
+    try:
+        method = ['conserve','bilinear','patch']
+        short = ['aave','blin','patc']
+        for j in range(len(method)):
```

Comment:
The more "pythonic" way to do this is:
```python 
for method, short in [['conserve', 'aave'], ['bilinear', 'blin'], ['patch', 'patc']]:
```
I did this as a list of lists but it could also be a `tuple` of `tuples` (with parentheses instead of brackets).

Then, you would drop the index `[j]` below.

Reply:
That's helpful. I changed it.